css/components/_alt-nav.scss

```diff
@@ -81,5 +143,39 @@
                 outline-color: $red;
             }
         }
+
+        li {
+            @include media("<desktop") {
+                margin: 0 0 8px;
+            }
+
+            &:last-child {
+                a {
+                    border-radius: 0 0 4px 4px;
+                }
+            }
+        }
+    }
+
+    &__overlay {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background: rgba($black, .2);
+        z-index: 101;
+        display: none;
+        pointer-events: none;
+
+        @include media(">=desktop") {
+            display: none !important;
+        }
```

Comment:
Why do we need `important` here?
Also, it duplicates `display: none;` above.

Reply:
js will add inline display block on some resolution